Brewfile

```diff
@@ -1,12 +1,9 @@
-cask_args appdir: "/Applications"
-tap "homebrew/cask"
-brew "terraform"
-brew "terragrunt"
-brew "kubernetes-cli"
-brew "kubernetes-helm"
-brew "aws-iam-authenticator"
-brew "azure-cli"
-brew "dnsmasq"
-brew "go"
-
-cask "aws-vault"
```

Comment:
Replacing the `aws-vault` cask with a formula doesn't appear to work on macOS.

Reply:
Looks like y'all need multiple OS's for CI testing :P 

I've updated it more to have some more logic, but I'm not thrilled with having active scripts in a file that's just supposed to list packages. 

I've used scripts in the brewfile in the past with no issues, but it feels like bad practice.  

An alternative would be to have separate OS specific Brewfiles and include a note in the readme to use a file path flag.